plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -654,7 +654,7 @@ func podMatchesNodeLabels(pod *v1.Pod, node *v1.Node) bool {
 	return nodeAffinityMatches
 }
 
-func PodSelectorMatches(pod *v1.Pod, meta interface{}, nodeInfo *schedulercache.NodeInfo) (bool, []algorithm.PredicateFailureReason, error) {
+func PodMatchNodeSelector(pod *v1.Pod, meta interface{}, nodeInfo *schedulercache.NodeInfo) (bool, []algorithm.PredicateFailureReason, error) {
```

Comment:
Sure, I will make a follow up PR for this.

@timothysc can you please add milestone 1.7 for this so that it can be in the submit queue?